tests/test_aio.py

```diff
@@ -1,7 +1,12 @@
 import asyncio
+import sys
 
 from livekit.agents.utils import aio
 
+# https://github.com/saghul/aiodns/issues/86
+if sys.platform == "win32":
+    asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
+
```

Comment:
It seems like it is only fixing the tests?
Real Windows users will actually face this issue.

Reply:
this is an issue to be fixed on the user side, not us. if they use asyncio anywhere (like the tests do before getting a worker) it's going to fail anyway

Reply:
the event loops are configured by our framework (one for the worker and one per job). So this isn't something user can fix.

I'm wondering what is the best solution here:
 - What are the tradeoffs of using the WindowsSelectorEventLoopPolicy on Windows? 
 - Otherwise we can just disable aiodns for Windows

Reply:
disabled aiodns on windows: the tests seem to pass but i'm not convinced they actually do, there's some errors that get ignored